metacluster/include/metacluster/CreateTenant.actor.h

```diff
@@ -157,10 +160,15 @@ struct CreateTenantImpl {
 		if (!self->assignClusterAutomatically) {
 			DataClusterMetadata dataClusterMetadata =
 			    wait(getClusterTransaction(tr, self->tenantEntry.assignedCluster));
-			if (!dataClusterMetadata.entry.hasCapacity()) {
+			if (!dataClusterMetadata.entry.hasCapacity() && !self->ignoreCapacityLimit) {
 				throw cluster_no_capacity();
 			}
 			dataClusterNames.push_back(self->tenantEntry.assignedCluster);
+		} else if (self->ignoreCapacityLimit) {
```

Comment:
I think we've already ruled out this case in the `run` function

Reply:
Done